src/main/java/com/example/DataService/CurrencyService.java

```diff
@@ -0,0 +1,36 @@
+package com.example.DataService;
+
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import com.example.repository.CurrencyRepository;
+
+import model.Asset;
+import model.Currency;
+
+@Service
+public class CurrencyService {
+	
+	private CurrencyRepository cr;
+	
+	public CurrencyService(CurrencyRepository cr) {
+		this.cr = cr;
+	}
+	
+	public void putCurrencies(){
+
+		final String uri = "https://api.coincap.io/v2/assets/";
+	    RestTemplate restTemplate = new RestTemplate();
+	    Asset result = restTemplate.getForObject(uri, Asset.class);
+	    for(int i = 1; i <= 10; i++) {
```

Comment:
Do not use magic numbers. Reference: https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Reply:
What if the response does not contain 10 entries? It will break your code.